pldoc/static/js/edx-icons.js

```diff
@@ -1 +1 @@
-require('font-awesome-loader');
\ No newline at end of file
+require('font-awesome-loader');
```

Comment:
All js files have a new line in it. Otherwise, it won't pass ESlint. Not sure why GH doesn't show. If you click View ^^^, pldoc/static/js/edx-icons.js has 2 lines.